dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java

```diff
@@ -27,6 +27,8 @@
  * </ol>
  */
 public class JakartaEE11Action extends JakartaEEAction {
+    public static final String ID = EE11_ACTION_ID;
```

Comment:
I purposely didn't add this variable to force people to use the ones in JakartaEEAction to avoid calling methods / fields directly on JakartaEE11Action.

Reply:
I don't understand why